packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -95,6 +95,25 @@ pub struct ExperimentConfig {
     clap(global = true, short = 'w', long, default_value_t = num_cpus::get(), validator = at_least_one, env = "HASH_WORKERS")
     )]
     pub num_workers: usize,
+
+    #[cfg_attr(feature = "clap", clap(global = true, long, default_value_t = 0))]
```

Comment:
I'm still not 100% sure what this default should be.
According to [this reply](https://stackoverflow.com/a/58041256) this is purely about GC and not the heap size itself.
To me it seems fine to have GC do its job whenever by default.

Reply:
I would prefer to not overwrite this value by default at all.

Reply:
Me too but we can't set the max size without setting the initial size.
And setting the max size seems a good idea in general.

Reply:
Not setting default anymore https://github.com/hashintel/hash/pull/446/commits/39550930d1416cf96fc927fb51721b2c0c7c0f77